ros/src/fsds_ros_bridge/include/statistics.h

```diff
@@ -20,19 +20,16 @@ namespace ros_bridge
         void Print()
         {
             // print statistics summary to the console
-            std::cout << "-----------------------------------------------------\n";
```

Comment:
maybe the lines between messages are not required, but we should add lines between print outs for clarity. So at the end of the print function in the wrapper

Reply:
added